src/blueapi/utils/path_provider.py

```diff
@@ -18,14 +18,19 @@ class StartDocumentPathProvider(PathProvider):
 
     def __init__(self) -> None:
         self._doc = {}
+        self._uid = None
```

Comment:
Can we use `None` for the 'there is no document' case as well? Having an empty dict is odd